gnosis/safe-react
# Remove truffle dependency

## What it solves
We have truffle dependency that we aren't almost using, but truffle package it's a heavy package. The operation that we are solving with truffle can be done with other dependencies that we already have, like web3.
Currently adding truffle makes the build almost 3MB bigger on a production build

## How this PR fixes it
Change the two places where we are using truffle to use web3 instead

## How to test it
 - You should be able to execute an ERC20 transaction without issues
 - You should be able to execute an ERC721 (collectibles) transaction without issues
